pyvecorg/templates/index.html

```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="utf-8">
 
-        <link rel="shortcut icon" type="image/vnd.microsoft.icon" href="{{ url_for('static', filename='favicon.ico') }}">
+        <link rel="shortcut icon" type="{{ favicon_mediatype }}" href="{{ url_for('static', filename='favicon.ico') }}">
```

Comment:
this can be just image/vnd.microsoft.icon everywhere, can't it? It's not actually checked anywhere and if it not accurate MIME type when serving on macOS, it's not breaking anything.

Reply:
I can't freeze the site without this, because Elsa errors on media type mismatch.

Reply:
I meant only if it when creating the responce, but not here in the HTML tag.